sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -46,6 +46,14 @@ static const pi_uint32 ZeSerialize = [] {
   return SerializeModeValue;
 }();
 
+// This is an experimental option to test performance of device to device copy
+// operations on copy engines (versus compute engine)
+static const bool UseCopyEngineForD2DCopy = [] {
+  const char *CopyEngineForD2DCopy =
+      std::getenv("SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE_FOR_D2D_COPY");
```

Comment:
Shouldn't this environment variable be documented in https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md?

Reply:
This is a temporary variable that we have included to facilitate experimentation for performance analysis. We expect to replace this very soon with a heuristic based approach.

Reply:
Okay, but I think we still should follow the good practice and document this variable.
I don't know when the variable will be removed and we need to make sure it's doesn't leak into some product undocumented.

Reply:
Agree. You can mark it as (experimental). As soon as done, you can remove it from docs as well.

Reply:
What we really want here is that this env var is NOT USED by anyone except those who participate in tuning of heuristics. To me the best way to achieve this is to not document it (except in the sources).

Reply:
> What we really want here is that this env var is NOT USED by anyone except those who participate in tuning of heuristics.

Adding environment variable to open source project doesn't guarantee this.
Can developers participating in tuning of heuristics build DPC++ compiler locally? If so, we can avoid adding environment variable support for all DPC++ users.

Reply:
I added some documentation. Please comment on the wording here. Thanks

Reply:
I am OK adding doc saying it is temporary and is expected to be removed (so people can't rely on it)